bootstrapvz/common/fs/loopbackvolume.py

```diff
@@ -15,7 +15,7 @@ def _before_create(self, e):
         log_check_call(['truncate', size_opt, self.image_path])
 
     def _before_attach(self, e):
-        [self.loop_device_path] = log_check_call(['losetup', '--show', '--find', self.image_path])
+        [self.loop_device_path] = log_check_call(['losetup', '--show', '--find', '--partscan', self.image_path])
```

Comment:
I don't understand the change, how adding a `--partscan` will allow to unmap the partitions?